tests/ctest/external-messaging2-toit-run-input.toit

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2024 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import expect show *
+import monitor
+import rpc
+
+TYPE ::= 100  // Don't overlap with system messages.
```

Comment:
Maybe move this down as a static in the MessageHandler class? It isn't used for RPC.

Reply:
done.